app/views/users/articles_written.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>user.name 's' Articles</h1>
+<ul>
+  <% user.articles.each do |article| %>
+    <li>
+    <%= link_to article.title, article.link %> - Posted by <%= link_to article.user.name, article.user %> on <%= article.created_at.strftime("%B %d, %Y at %I:%M %p") %>
+    <div>
+      <%= link_to "comment", article %>
+    </div>
```

Comment:
These lines should be tabbed in since they are inside the `<li>` block.

Reply:
Didn't mean to push this.